src/modules/mana/selectors.ts

```diff
@@ -0,0 +1,39 @@
+import { getData as getTransactions } from 'decentraland-dapps/dist/modules/transaction/selectors'
+import { isPending } from 'decentraland-dapps/dist/modules/transaction/utils'
+import { getAddress } from 'decentraland-dapps/dist/modules/wallet/selectors'
+import { createSelector } from 'reselect'
+import {
+  Transaction,
+  TransactionStatus,
+  TransactionType,
+} from '../../components/HomePage/HomePage.types'
+import { RootState } from '../reducer'
+
+export const getState = (state: RootState) => state.mana
+export const getData = (state: RootState) => getState(state).data
+export const getLoading = (state: RootState) => getState(state).loading
+export const getError = (state: RootState) => getState(state).error
+export const getManaPrice = (state: RootState) => getData(state).price
+export const getManaTransaction = createSelector(
```

Comment:
we can remove this selector, since it's not being used for now (we will add it once we do the deposit mana feature)